### ISSUE TITLE
Different kernel sizes for different dimensions

### ISSUE THREAD

**Issue by kuangdai** [fepegar/unet#40]
Can you maybe add this feature, please? 

I think the main change would be 

* From

```python
class ConvolutionalBlock(nn.Module):
    def __init__(
            self,
            dimensions: int,
            in_channels: int,
            out_channels: int,
            normalization: Optional[str] = None,
            kernel_size: int = 3,
            activation: Optional[str] = 'ReLU',
            preactivation: bool = False,
            padding: int = 0,
            padding_mode: str = 'zeros',
            dilation: Optional[int] = None,
            dropout: float = 0,
            ):
```

* To

```python
class ConvolutionalBlock(nn.Module):
    def __init__(
            self,
            dimensions: int,
            in_channels: int,
            out_channels: int,
            normalization: Optional[str] = None,
            kernel_size: int = Union[int, Sequence[int]],
            activation: Optional[str] = 'ReLU',
            preactivation: bool = False,
            padding: int = 0,
            padding_mode: str = 'zeros',
            dilation: Optional[int] = None,
            dropout: float = 0,
            ):
```

I can create a PR if you prefer this way.

**Comment by fepegar** [comment]
Hi, @kuangdai. Please feel free to open the PR!

**Comment by lizhuoq** [comment]
If `kernel_size` can accept a tuple, doesn't it mean that `padding` should also accept a tuple?
